docs/troubleshooting.md

```diff
@@ -0,0 +1,105 @@
+# Troubleshooting
+
+## Debugging with the CLI
+
+The easiest way to debug your `release-please` configuration is to use the bundled
+CLI. 
+
+To use the CLI, you will need to use a GitHub personal access token:
+
+```bash
+export GITHUB_TOKEN="my-access-token"
+release-please release-pr \
```

Comment:
Since many folks using the tool are non JavaScript users, I would add a section on:

"Setting up your environment", which links to [nvm](https://github.com/nvm-sh/nvm) and explains they will need to:

1. git clone the repository.
2. setup Node.js with nvm.
3. npm i
4. npm link .

Reply:
I don't think it's necessary to teach folks how to install node.